Sources/MockableMacro/Generator/Helpers/FunctionDeclaration.swift

```diff
@@ -113,7 +113,9 @@ extension FunctionDeclaration {
     var filteredGenericParameterClause: GenericParameterClauseSyntax? {
         guard let generics = syntax.genericParameterClause else { return nil }
         var parameters = generics.parameters.filter { generic in
-            hasParameter(containing: generic.name.trimmedDescription)
+            let hasParameter = hasParameter(containing: generic.name.trimmedDescription)
+            let hasReturn = hasReturn(containing: generic.name.trimmedDescription)
```

Comment:
Return only generics is what this prop was meant to filter out.

Reply:
But, in case the generic is only on the return clause, will not pass filtering since there is no generic parameter matching, right?